server/cmd/mmctl/commands/import.go

```diff
@@ -95,7 +95,9 @@ var ImportValidateCmd = &cobra.Command{
 	Example: "  import validate import_file.zip --team myteam --team myotherteam",
 	Short:   "Validate an import file",
 	Args:    cobra.ExactArgs(1),
-	RunE:    importValidateCmdF,
+	RunE: func(command *cobra.Command, args []string) error {
+		return importValidateCmdF(nil, command, args)
+	},
```

Comment:
What is the reason for doing it like this?

Reply:
Testing mainly. The command is designed to work either with or without a client but in unit tests I wanted to actually test with a mocked client that the returned config values (`MaxPostSize`) are used as expected.